test/mpi/util/mtest_thread_abt.h

```diff
@@ -47,6 +47,11 @@ static void MTest_abt_error(int err, const char *msg, const char *file, int line
 
 #define MTEST_NUM_XSTREAMS 2
 
+/* Reference counter */
+
+static void MTest_abt_incr_refcount(void);
+static void MTest_abt_decr_refcount(void);
```

Comment:
I understand this is only included in `mtest_thread.c`, but it is concerning to have static implementation inside a `.h` header.

Reply:
In terms of coding practice, I believe non-static/inline definitions should be avoided since it often causes a "multiple definition" error.

In this specific case, static means these functions cannot be called by individual tests.